jaxrs-clients/versions.lock

```diff
@@ -189,6 +203,25 @@
                 "com.palantir.remoting-api:service-config"
             ]
         },
+        "com.palantir.tritium:tritium-api": {
+            "locked": "0.7.0",
+            "transitive": [
+                "com.palantir.tritium:tritium-core",
+                "com.palantir.tritium:tritium-metrics"
+            ]
+        },
+        "com.palantir.tritium:tritium-core": {
```

Comment:
what's the point of tritium-api if I need to depend on this whole transitive beast?